graphics/src/window/compositor.rs

```diff
@@ -49,5 +52,5 @@ pub trait Compositor: Sized {
         background_color: Color,
         output: &<Self::Renderer as iced_native::Renderer>::Output,
         overlay: &[T],
-    ) -> mouse::Interaction;
+    ) -> Result<mouse::Interaction, ()>;
```

Comment:
I think a proper error type here would make sense, maybe just copy `SwapChainError`.

Reply:
Maybe? I think that the `SwapChainError` is only specific to `wgpu` though.

Reply:
Actually, yeah that could work. I've made those changes.